scripts/since-latest-release.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+magenta() {
+  printf "\x1B[35m"
+  echo -n "$@"
+  printf "\x1B[0m"
+}
+
+bold() {
+  printf "\x1B[1m"
+  echo -n "$@"
+  printf "\x1B[22m"
+}
+
+previous-release-commit() {
+  git tag --sort=version:refname --list "$1@*" | tail -n 1
+}
+
+determine-commit-range() {
+  if [[ "$1" =~ ^release/ ]]; then
+    echo "$previous_release_commit..main"
```

Comment:
This makes it so that if you're on a release branch, the list of changes you see don't include any of the commits in the release branch itself.